the-telephone-app-fe/src/components/StartGame/StartGame.js

```diff
@@ -75,6 +75,7 @@ const StartGame = () => {
         navigate("/book", {
           state: { data, id_room, role, name, dataReceive, turn },
         });
+        setTurn(0);
```

Comment:
I mean for the above code the value for turn is assigned directly and setTurn haven't been used, so is useState needed for it? Can it be just 'let turn =1' instead of 'let [turn, setTurn] = useState(1);' and still work properly?